README.md

```diff
@@ -33,7 +33,26 @@ export KUBE_TEST_REPO_LIST=$(pwd)/repo_list
 
 ### e2e.test
 
-> TODO
+Make sure you have a working [Kubernetes development environment](https://github.com/kubernetes/kubernetes).
```

Comment:
https://github.com/kubernetes/community/blob/master/contributors/devel/development.md has better steps on the dev environment, can you link to that?